pages.pt-BR/windows/cls.md

```diff
@@ -0,0 +1,7 @@
+# cls
+
+> Limpa a tela de saída.
+
+- Limpa a tela:
```

Comment:
Please use infinitive as in the other pages: "Limpar". (Same in the main description above)